documentation/src/docs/asciidoc/user-guide/appendix.adoc

```diff
@@ -199,6 +208,12 @@ runner ..> launcher
 runner ..> suite_api
 runner ..> junit4
 
+suite ..> suite_api
+suite ..> suite_engine
+
+suite_engine ..> launcher
+suite_engine ..> suite_api
```

Comment:
This image is becoming quite crowded now. I'm not sure how to improve it.

Reply:
By adding some more space (additional dots) between the connections that cross package boundaries graphviz seems to be able to sort it out again.